src/renderer/windows/GPMWebView/playback/controller.js

```diff
@@ -76,3 +76,11 @@ Emitter.on('playback:miniEnable', () => {
 Emitter.on('playback:miniDisable', () => {
   window.GPM.mini.disable();
 });
+
+Emitter.on('playback:infoTrack', () => {
+  if (!remote.getGlobal('PlaybackAPI').data.song.title) return;
+  new Notification(remote.getGlobal('PlaybackAPI').data.song.title, { // eslint-disable-line
```

Comment:
These remote calls are taxxing on the UI and not required.  The information we need is available in this process through the `GPM` global.

https://github.com/gmusic-utils/gmusic.js#playbackgetcurrenttrack

```js
GPM.getCurrentTrack();
``